src/foam/u2/wizard/StepWizardletController.js

```diff
@@ -98,6 +98,33 @@ foam.CLASS({
         return sections.reduce(
           (sum, wizardletSections) => sum + wizardletSections.length, 0);
       }
+    },
+    {
+      name: 'canGoBack',
+      class: 'Boolean',
+      documentation: `
+        If the first screen has no available sections, then the back button
+        should be disabled.
+      `,
+      expression: function (subStack$pos) {
+        let indices = this.screenIndexToSection(subStack$pos);
```

Comment:
screenIndexToSection(...) can return null - what are you doing to handle this call - not only here but everywhere?

Reply:
Should never happen (requires invalid subStack position) but I'll modify screenIndexToSection to throw an error if the stack position isn't okay.